storage-operator/pkg/apis/storage/v1alpha1/volume_types.go

```diff
@@ -63,44 +63,72 @@ type PersistentVolumeTemplateSpec struct {
 
 type VolumePhase string
 
-// "Enum" representing the phase/status of a volume.
+// TODO: kept for temporary compatibility, to be removed.
```

Comment:
Temporary? What would condition the end of this temporary time?

Reply:
When we will have fine-grained conditions representing each steps, the concept of global phase may become irrelevant/useless and thus it could be removed (some k8s objects only have conditions).
I'll try to open a new issue about this, containing the more completed design I first proposed for this one (but which I then simplified due to the time constraint)

Reply:
OK thx for the explanation.